src/App.js

```diff
@@ -12,29 +12,80 @@ import SignUp from "./pages/SignUp";
 import ActivityEdit from "./pages/ActivityEdit";
 import ActivityFilter from "./pages/ActivityFilter";
 import AboutUs from "./pages/AboutUs";
+import mockUsers from "./mockUsers";
 
 
 function App() {
-const [activities, setActivities] = useState(mockActivities)
+const [activities, setActivities] = useState([])
+const [currentUser, setCurrentUser] = useState(mockUsers[0])
+
 
 const [users, setUsers] = useState(mockUsers)
 
 //*******This is for testing with mock data. DELETE THIS AFTER BACK END IS CONNECTED*******
+// const createActivity = (activity) => {
+//   console.log(activity)
+// }
+// const updateActivity = (activity) => {
+//   console.log(activity)
+// }
+
+// const deleteActivity = (activity) => {
+//   console.log(activity)
+// }
+
+  useEffect(() => {
+    readActivity()
+  }, [])
+
+  const readActivity = () => {
+    fetch("http://localhost:3000/activities")
```

Comment:
Down the road you will need to update the url to the deployed url for production.

Reply:
If that's just the name of our web address, or the internal render address, I'm happy to do that in the morning before we merge.